src/app/courses/add-courses/courses-add.scss

```diff
@@ -1,4 +1,5 @@
 @import '../../variables';
+@import '../../../styles.scss';
```

Comment:
@sjson421 You don't actually need to import the `styles.scss` files because that css is included broadly as a sort of global css.

Reply:
@paulbert Hmm I don't why why my IDE yelled at me there. Fixed!